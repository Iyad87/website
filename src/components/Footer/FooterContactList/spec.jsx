import React from 'react';
import { shallow } from 'enzyme';
import FooterContactList from './index';

test('components/FooterContactList snapshot', () => {
  const wrapper = shallow(<FooterContactList />);

  expect(wrapper).toMatchSnapshot();
});
