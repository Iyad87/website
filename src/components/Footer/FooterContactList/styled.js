import styled from 'styled-components';

import DefaultLink from 'components/Link';
import { footerText } from 'styles/colours';
import { spacing } from 'styles/spacing';
import breakpoints from 'styles/breakpoints';

export const Contact = styled.li`
  padding: ${spacing(2)};
  width: 50%;

  ${breakpoints.tablet`
    margin-bottom: 0;
    width: ${100 / 3}%;
  `};
`;

export const List = styled.ul`
  display: flex;
  flex-wrap: wrap;
  list-style-type: none;
  margin: -${spacing(2)};
  padding: 0;
`;

export const Line = styled.p`
  margin-bottom: ${spacing()};
  margin-top: 0;
  font-size: 1.125rem;
  font-style: normal;
`;

export const Link = styled(DefaultLink)`
  color: ${footerText};
  font-weight: 400;
`;

export const Office = Line.withComponent('h6').extend`
  font-weight: 500;
  margin-bottom: ${spacing(2)};
`;
