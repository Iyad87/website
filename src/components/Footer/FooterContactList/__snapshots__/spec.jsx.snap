// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/FooterContactList snapshot 1`] = `
<styled.ul>
  <styled.li>
    <address>
      <styled.h6>
        London
      </styled.h6>
      <styled.p>
        124 Aldersgate Street
      </styled.p>
      <styled.p>
        EC1A 4JQ
      </styled.p>
      <styled.p>
        <Styled(Link)
          href="tel:+44-203-514-4678"
        >
          +44 (0) 203 514 4678
        </Styled(Link)>
      </styled.p>
      <styled.p>
        <Styled(Link)
          href="mailto:hello@yld.io"
        >
          hello@yld.io
        </Styled(Link)>
      </styled.p>
    </address>
  </styled.li>
  <styled.li>
    <address>
      <styled.h6>
        Lisbon
      </styled.h6>
      <styled.p>
        Rua Ramalho Ortigão 8
      </styled.p>
      <styled.p>
        3rd floor left
      </styled.p>
      <styled.p>
        1070-230
      </styled.p>
    </address>
  </styled.li>
  <styled.li>
    <address>
      <styled.h6>
        Porto
      </styled.h6>
      <styled.p>
        Rua Ramalho Ortigão 8
      </styled.p>
      <styled.p>
        3rd floor left
      </styled.p>
      <styled.p>
        1070-230
      </styled.p>
    </address>
  </styled.li>
</styled.ul>
`;
